game/scripts/npc/abilities/phantom_lancer_spirit_lance.txt

```diff
@@ -19,11 +19,11 @@
     "MaxLevel"                                            "6"
     // Casting
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCastPoint"                                    "0.3 0.3 0.3 0.3"
+    "AbilityCastPoint"                                    "0.3"
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "7.0 7.0 7.0 7.0"
+    "AbilityCooldown"                                     "9.0" //OAA
```

Comment:
This is on purpose because of the Aghs and duration at lvl 6.

Reply:
Reduce the duration at level 6 then. Don't nerf the heroes entire game because of an item which most PL players won't buy and because at max level the duration is too long.